src/append/stdio.rs

```diff
@@ -36,8 +35,7 @@ pub struct Stderr;
 
 impl Append for Stderr {
     fn append(&self, record: &log::Record) -> anyhow::Result<()> {
-        let bytes = format!("{}\n", record.args()).into_bytes();
-        std::io::stderr().write_all(&bytes)?;
+        writeln!(std::io::stderr(), "{}", record.args())?;
```

Comment:
I don't check in details but IIRC fern has some related tricks to avoid deadline in logging.

Check - https://docs.rs/fern/0.6.2/fern/meta/index.html

And @Xuanwo ever encountered similar situation (dead lock in fmt).

Reply:
And we only call format to alloc the string then write the bytes. IIRC it won't alloc twice?

Reply:
it make sense